token-lending/cli/src/main.rs

```diff
@@ -0,0 +1,700 @@
+    );
+    create_accounts_transaction_2.sign(
+        &vec![
+            config.fee_payer.as_ref(),
+            &liquidity_supply_keypair,
+            &liquidity_fee_receiver_keypair,
+            &liquidity_host_keypair,
+        ],
+        recent_blockhash,
+    );
+    init_reserve_transaction.sign(
+        &vec![
+            config.fee_payer.as_ref(),
+            &lending_market_owner_keypair,
+            &user_transfer_authority_keypair,
+        ],
+        recent_blockhash,
+    );
+    send_transaction(&config, create_accounts_transaction_1)?;
+    send_transaction(&config, create_accounts_transaction_2)?;
+    send_transaction(&config, init_reserve_transaction)?;
+    Ok(())
+}
+
+fn main() {
+    solana_logger::setup_with_default("solana=info");
+
+    let matches = App::new(crate_name!())
+        .about(crate_description!())
+        .version(crate_version!())
+        .setting(AppSettings::SubcommandRequiredElseHelp)
+        .arg({
+            let arg = Arg::with_name("config_file")
+                .short("C")
+                .long("config")
+                .value_name("PATH")
+                .takes_value(true)
+                .global(true)
+                .help("Configuration file to use");
+            if let Some(ref config_file) = *solana_cli_config::CONFIG_FILE {
+                arg.default_value(&config_file)
+            } else {
+                arg
+            }
+        })
+        .arg(
+            Arg::with_name("verbose")
+                .long("verbose")
+                .short("v")
+                .takes_value(false)
+                .global(true)
+                .help("Show additional information"),
+        )
+        .arg(
+            Arg::with_name("dry_run")
+                .long("dry-run")
+                .takes_value(false)
+                .global(true)
+                .help("Simulate transaction instead of executing"),
+        )
+        .arg(
+            Arg::with_name("json_rpc_url")
+                .long("url")
+                .value_name("URL")
+                .takes_value(true)
+                .validator(is_url)
+                .help("JSON RPC URL for the cluster.  Default from the configuration file."),
+        )
+        .arg(
+            fee_payer_arg()
+                .short("p")
+                .global(true)
+        )
+        .subcommand(
+            SubCommand::with_name("create-market")
+                .about("Create a new lending market")
+                .arg(
+                    Arg::with_name("lending_market_owner")
+                        .long("owner")
+                        .validator(is_pubkey)
+                        .value_name("PUBKEY")
+                        .takes_value(true)
+                        .required(true)
+                        .help("Owner required to sign when adding reserves to the lending market"),
+                )
+                .arg(
+                    Arg::with_name("oracle_program_id")
```

Comment:
It would be nice to have a default value for this

Reply:
Done.